core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -304,6 +328,15 @@ angular.module('oppia').factory('ResponsesService', [
         var oldAnswerChoices = angular.copy(_answerChoices);
         _answerChoices = newAnswerChoices;
 
+        // If the interaction is MultipleChoiceInput, update the answer groups
+        // to refer to the new answer options.
+        if (StateInteractionIdService.savedMemento === 'MultipleChoiceInput') {
+          if (_answerGroupsMemento &&
```

Comment:
It seems like this is being handled differently from ItemSelectionInput below. Can we do something here that mimics what is already being done there, instead? That would probably be a better design since it keeps everything consistent.

Reply:
done